src/fklearn/metrics/pd_extractors.py

```diff
@@ -19,16 +19,19 @@ def combined_evaluator_extractor(result, base_extractors):
 
 
 @curry
-def split_evaluator_extractor_iteration(split_value, result, split_col, base_extractor):
-    key = 'split_evaluator__' + split_col + '_' + str(split_value)
+def split_evaluator_extractor_iteration(split_value, result, eval_name, base_extractor):
+    key = eval_name + '_' + str(split_value)
     return (base_extractor(result.get(key, {}))
-            .assign(**{'split_evaluator__' + split_col: split_value}))
+            .assign(**{eval_name: split_value}))
```

Comment:
I don't think we should break the api(as you pointed in the description) to fix this bug, I will try to think about ways to solve it 🤔 

I think we are missing tests for this, so this is the reason why everything succeeded

Reply:
hmmm I was split_evaluator_extractor_iteration as a private function